### PR TITLE
packaging: display First Install and Onboarding Message

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR displays some cloudstack information when install cloudstack packages and on UI.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: scripts/installer/cloudstack-help-text [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# specific language governing permissions and limitations
+# under the License.
+
+printf "\n"
+printf "Thank you for installing Apache CloudStack !\n"
+printf "\n"
+printf " * Project:          https://cloudstack.apache.org/\n"
+printf " * Documentation:    https://docs.cloudstack.apache.org/\n"
+printf " * Join us:          https://cloudstack.apache.org/mailing-lists.html\n"
+printf " * Your feedback:    https://cloudstack.apache.org/survey.html\n"
```
Comment (rohityadavcloud) [inline scripts/installer/cloudstack-help-text:25]: Let's write it as `Take the Survey`?

File: scripts/installer/cloudstack-help-text [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+printf " * Report issues:    https://github.com/apache/cloudstack/issues\n"
+printf " * Latest releases:  https://github.com/apache/cloudstack/releases\n"
```
Comment (rohityadavcloud) [inline scripts/installer/cloudstack-help-text:27]: Not sure about this line - don't know if this is useful?

File: scripts/installer/cloudstack-help-text [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+printf "\n"
+printf "Thank you for installing Apache CloudStack !\n"
+printf "\n"
+printf " * Project:          https://cloudstack.apache.org/\n"
```
Comment (rohityadavcloud) [inline scripts/installer/cloudstack-help-text:22]: Let's call it `Project Website`?

File: scripts/installer/cloudstack-help-text [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+printf " * Documentation:    https://docs.cloudstack.apache.org/\n"
```
Comment (rohityadavcloud) [inline scripts/installer/cloudstack-help-text:23]: After this line, should we add the link to release notes of the current release/version, for example:
```
http://docs.cloudstack.apache.org/en/${$ACS_VERSION:-"latest"}/releasenotes/index.html
```

Comment (weizhouapache) [reply scripts/installer/cloudstack-help-text:23]: @rohityadavcloud good idea. I will add it

File: scripts/installer/cloudstack-help-text (hunk 3 above)
Comment (weizhouapache) [reply scripts/installer/cloudstack-help-text:22]: OK. I will update it

File: scripts/installer/cloudstack-help-text (hunk 2 above)
Comment (weizhouapache) [reply scripts/installer/cloudstack-help-text:27]: I will remove it then.

File: scripts/installer/cloudstack-help-text (hunk 1 above)
Comment (weizhouapache) [reply scripts/installer/cloudstack-help-text:25]: I will update it

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by sureshanaparti** [comment]
@blueorangutan test

**Comment by weizhouapache** [comment]
tested in a ubuntu20 docker container

image

image

I will test in other containers with different OS.

**Comment by weizhouapache** [comment]
centos7

(please ignore the errors which are caused by systemd)
image

image

**Comment by weizhouapache** [comment]
centos8

image

image

**Comment by weizhouapache** [comment]
suse15

image

image

**Comment by weizhouapache** [comment]
@rohityadavcloud @sureshanaparti @DaanHoogland 
this is ready for review

**Review by DaanHoogland — APPROVED** [review]

**Review by rohityadavcloud — APPROVED** [review]

**Comment by weizhouapache** [comment]
PR updated. now links of installed cloudstack version are displayed on UI.

image

this is ready for QA testing.

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by rohityadavcloud** [comment]
@blueorangutan test

**Review by borisstoyanov — APPROVED** [review]
LGTM, manually checked. All messages appear as suggested.

**Comment by weizhouapache** [comment]
thanks @borisstoyanov for testing !

**Pull request merged**